drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -50,6 +50,34 @@
                    class="coord y"
                    value="{{ y }}" />
         </div>
+        <div class="alignment">
+            <label for="zone-{{index}}-align">
+                {{i18n "Alignment"}}
+            </label>
+            <select id="zone-{{index}}-align"
+                   class="align-select"
+                   aria-describedby="zone-align-description" >
+                <option value="" 
+                    {{#ifeq align "" }}selected{{/ifeq}}>
+                    {{i18n "none"}}
+                </option>
+                <option value="left" 
+                    {{#ifeq align "left" }}selected{{/ifeq}}>
+                    {{i18n "left" }}
+                </option>
+                <option value="center" 
+                    {{#ifeq align "center" }}selected{{/ifeq}}>
+                    {{i18n "center" }}
+                </option>
+                <option value="right"
+                    {{#ifeq align "right" }}selected{{/ifeq}}>
+                    {{i18n "right" }}
+                </option>
+            </select>
+            <div id="zone-align-description" class="zones-form-help">
+                {{i18n "Align dropped items to the left, center, or right.  Default is no alignment."}}
```

Comment:
Perhaps clarify "Default is no alignment (items stay exactly where the user drops them)" ?